share/output_wrf.F

```diff
@@ -810,8 +836,13 @@ SUBROUTINE output_wrf ( fid , grid , config_flags, switch , ierr )
 #endif
 
 #if (EM_CORE == 1) && ( DA_CORE != 1)
+!!! Expanded to include auxiliary history files.  TLS 20 Jan 2021
+!!!      IF ( ( switch .EQ. input_only   ) .OR. &
+!!!           ( switch .EQ. history_only ) .OR. &
+!!!           ( switch .EQ. restart_only ) ) THEN
```

Comment:
I'd suggest to remove these commented lines. One can retrieve old code fairly easily with git.

Reply:
Thank you, @weiwangncar!

Would you like me to remove my comments that include the nature of the change, e.g., L839 in the example above?

Reply:
@tlspero I'd say we can remove that too. Making the purpose of your change clear in the commit message is sufficient. Thanks!